tuxemon/technique.py

```diff
@@ -86,13 +86,16 @@ class Technique:
             For example, monster that obtains life with lifeleech.
 
     """
+
     def __init__(
         self,
         slug: Optional[str] = None,
         carrier: Optional[Monster] = None,
         link: Optional[Monster] = None,
     ) -> None:
-        self._combat_counter = 0  # number of turns that this technique has been active
+        self._combat_counter = (
```

Comment:
Really?

Reply:
black does this with inline comments.  ive been removing them, or moving them to the line before the code to avoid this